venv/lib/python3.9/site-packages/pip/_internal/index/collector.py

```
@@ -0,0 +1,556 @@
+class _NotHTTP(Exception):
+    pass
+
+
+def _ensure_html_response(url, session):
+    # type: (str, PipSession) -> None
+    """Send a HEAD request to the URL, and ensure the response contains HTML.
+
+    Raises `_NotHTTP` if the URL is not available for a HEAD request, or
+    `_NotHTML` if the content type is not text/html.
+    """
+    scheme, netloc, path, query, fragment = urllib.parse.urlsplit(url)
```

Ruff (0.14.8)

Unpacked variable `netloc` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)